src/main/java/fr/openent/competences/helper/DateHelper.java

```diff
@@ -0,0 +1,44 @@
+package fr.openent.competences.helper;
+
+import fr.openent.competences.constants.DateFormat;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateHelper {
+    private static final Logger log = LoggerFactory.getLogger(DateHelper.class);
+
+    private DateHelper() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static Date parse(String dateString, String format) {
+        Date date = null;
```

Comment:
you are sure you want to use the legacy `Date` instead of `LocalDateTime` or `ZonedDateTime`?